warehouse/query-core/src/test/java/datawave/query/jexl/DatawaveInterpreterTest.java

```diff
@@ -186,6 +186,83 @@ public void testBoundedRangeOrTerm() {
         test(query, buildBoundedRangeContext(), false);
     }
     
+    // test filter:isNull
+    @Test
+    public void testFilterFunctionIsNull() {
+        // single field, present
+        String query = "FOO == 'bar' && filter:isNull(FOO)";
+        test(query, buildDefaultContext(), false);
+        
+        // single field, absent
+        query = "FOO == 'bar' && filter:isNull(ABSENT)";
+        test(query, buildDefaultContext(), true);
+        
+        // multi field, all present
+        query = "FOO == 'bar' && filter:isNull(FOO || FOO)";
+        test(query, buildDefaultContext(), false);
+        
+        // multi field, (present || absent)
+        query = "FOO == 'bar' && filter:isNull(FOO || ABSENT)";
+        // test(query, buildDefaultContext(), true); // this is wrong
```

Comment:
If these are wrong, maybe correct or remove them

Reply:
Moved the multi-fielded instances of `isNull` and `isNotNull` to their own test methods and put them behind an `@Ignore` annotation until #1604 can be worked.